website/content/docs/getting-started.md

```diff
@@ -39,14 +39,14 @@ Heron client installer
 Uncompressing......
 Heron is now installed!
 
-Make sure you have "/usr/local/bin" in your path.
+Make sure you have "~/bin" in your path.
```

Comment:
Heron when installed - actually prints the full path of '~' - should we change to ${HOME}?